packages/layout/src/steps/resolvePagination.js

```diff
@@ -260,6 +260,11 @@ const resolvePagination = (doc, fontStore) => {
 
   for (let i = 0; i < doc.children.length; i += 1) {
     const page = doc.children[i];
+    if (!page.props?.wrap) {
```

Comment:
We should explicitely check for false here. Otherwise missing `wrap` prop would prevent pages from wrapping which isn't what we want